deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -562,3 +564,151 @@ def predict_proba_on_generator(self, generator, transformers=[]):
             result = undo_transforms(result, transformers)
           results.append(result)
         return np.concatenate(results, axis=0)
+
+
+class GraphConvTensorGraph(TensorGraph):
+
+  def __init__(self, n_tasks, **kwargs):
+    """
+        Parameters
+        ----------
+        n_tasks: int
+          Number of tasks
+
+    """
+    self.n_tasks = n_tasks
+    kwargs['use_queue'] = False
+    super(GraphConvTensorGraph, self).__init__(**kwargs)
+    self.build_graph()
+
+  def build_graph(self):
+    """
+    Building graph structures:
+    """
+    self.atom_features = Feature(shape=(None, 75))
+    self.degree_slice = Feature(shape=(None, 2), dtype=tf.int32)
+    self.membership = Feature(shape=(None,), dtype=tf.int32)
+
+    self.deg_adjs = []
+    for i in range(0, 10 + 1):
+      deg_adj = Feature(shape=(None, i + 1), dtype=tf.int32)
+      self.deg_adjs.append(deg_adj)
+    gc1 = GraphConv(
```

Comment:
It would be nice to allow `**kwargs` control the architecture (say allow users to specify number of graph conv and graph gather layers and their ordering).

This could be done in a future PR though.